python/langsmith/schemas.py

```diff
@@ -74,6 +76,7 @@ class ExampleBase(BaseModel):
     inputs: Dict[str, Any] = Field(default_factory=dict)
     outputs: Optional[Dict[str, Any]] = Field(default=None)
     metadata: Optional[Dict[str, Any]] = Field(default=None)
+    attachments: Optional[Dict[str, Tuple[str, Callable[[], requests.Response]]]] = Field(default=None)
```

Comment:
Can we add a comment here to explain this type? It's pretty cumbersome. Also I don't think we need to be augmenting the type of examples here? And the typing certainly shouldn't be requests.Response